### ISSUE TITLE
Cannot restart NVDA after unloading it in XP and Windows 7

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1301]
**Reported by Rughead on 2011-01-05 04:40**
On both my XP Pro (SP3) and my Windows 7 computer when I quit NVDA (Insert+Q) I cannot restart it again. Note, at this point, I cannot start JAWS either. If I go to the Task Manager, and then go to Processes, NVDA is still listed there. If I terminate the NVDA process there, then, and only then, I can restart NVDA or JAWS. Of course, I needed a sighted person at this point to do this. This only happens with NVDA 10.2 release version (latest). Without a sighted person to help, I am left with no other choice other than to restart the computer.

**Comment by dkager** [comment]
Probably OK to close this one because the software involved has changed quite a bit since thereport and no new mention was made of it since 2011.
